eventuals/event-loop.h

```diff
@@ -841,9 +851,13 @@ inline auto& Clock() {
 ////////////////////////////////////////////////////////////////////////
 
 inline std::chrono::nanoseconds EventLoop::Clock::Now() {
+  CHECK(!loop_.running_);
```

Comment:
Why can't the loop be running?

Reply:
The time inside uv_loop is stored as uint64_t variable and is updated every iteration of uv_run. Its update is done non-atomically, hence trying to read the clock while running an EventLoop in another thread is UB and will return an incorrect result.

Reply:
I'm still not following. Based on your concerns it seems like we should be able to do `Now()` at the very least if we're `InEventLoop()`, which is exactly when most libuv users would likely call `uv_now()` themselves.

Given this `CHECK()` the only time you could really use `Now()` is in testing before the event loop has started, which might be what triggered you to make this change but it's definitely not how most users are going to want to use `Now()` when the event loop is _always_ running.

Reply:
Updated!

Reply:
Did you push the new code? I'm still seeing the old code it seems like.